src/main/java/frc/lib/controller/PIDGains.java

```diff
@@ -9,7 +9,7 @@ public PIDGains(double p, double i, double d) {
 
```

Comment:
Let's move all these to the Constants class.  Make sure to name it something related to game element lock on.

Reply:
This Class is part of the main branch?